phoenix-core/src/main/java/org/apache/phoenix/compile/FromCompiler.java

```diff
@@ -893,15 +893,14 @@ public Void visit(NamedTableNode tableNode) throws SQLException {
             TableRef tableRef = createTableRef(connectionSchemaName, tableNode, true, false);
             PTable theTable = tableRef.getTable();
 
+            String name = theTable.getName().getString();
```

Comment:
nit: maybe extract this logic to a private helper function so it doesn't have to be duplicated in two places

Reply:
Done.